src/firebolt/async_db/connection.py

```diff
@@ -231,10 +233,22 @@ async def connect(
 
     else:
         try:
+            timerStart = time()
             engine_url, status, attached_db = await _get_engine_url_status_db(
                 system_engine_connection, engine_name
             )
 
+            teimerEnd = time()
+            envVar = "0"
+            try:
+                envVar = environ["FIREBOLT_SDK_PERFORMANCE_DEBUG"]
+            except Exception:
+                pass
```

Comment:
Can we replace this logic with `environ.get("FIREBOLT_SDK_PERFORMANCE_DEBUG", "0")`?